test/dock_test.rb

```diff
@@ -0,0 +1,75 @@
+require "minitest/autorun"
+require "minitest/pride"
+require "./lib/dock"
+require "./lib/renter"
+require "./lib/boat"
+
+class DockTest < Minitest::Test
+  def test_it_exists
+    dock = Dock.new("The Rowing Dock", 3)
+    assert_instance_of Dock, dock
+  end
+
+  def test_dock_has_a_name
+    dock = Dock.new("The Rowing Dock", 3)
+
+    assert_equal "The Rowing Dock", dock.name
+  end
+
+  def test_dock_has_max_rental_time
+
+    dock = Dock.new("The Rowing Dock", 3)
+
+    assert_equal 3, dock.max_rental_time
+  end
+
+  def test_dock_can_have_boat_and_boats_can_have_renters
+    dock = Dock.new("The Rowing Dock", 3)
+    ringo = Renter.new("Ringo Star", "4242424242424242")
+    kayak_1 = Boat.new(:kayak, 20)
+    dock.rent(kayak_1, ringo)
+
+    assert_instance_of Renter, dock.renters.last
+    assert_instance_of Boat, dock.boats.last
```

Comment:
You shouldn't just assert it's a boat, you sure assert the EXACT boat. You have access to it. It's called `kayak_1`. Same thing for `ringo`